### ISSUE TITLE
Error when using combined option

### ISSUE THREAD

**Issue by majkinetor** [zhaoterryy/mkdocs-pdf-export-plugin#39]
Everything works fine when seen individually, but combined I miss certain pages without error. I moved one of those missing pages to be the first one in nav list and then it errored with `AttributeError: 'NoneType' object has no attribute 'copy'`

```
/usr/lib/python3.6/site-packages/weasyprint/document.py:34: UserWarning: There are known rendering problems and missing features with cairo < 1.15.4. WeasyPrint may work with older versions, but please read the note about the needed cairo version on the "Install" page of the documentation before reporting bugs. http://weasyprint.readthedocs.io/en/latest/install.html
  'There are known rendering problems and missing features with '
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: /docs/source/site
ERROR: Failed to load font at "file:///docs/source/site/assets/fonts/specimen/MaterialIcons-Regular.woff2"
Error converting demo.md to PDF: Invalid ID found in demo/, ID: fnref:short
ERROR: Failed to load font at "file:///docs/source/site/assets/fonts/specimen/MaterialIcons-Regular.woff2"
Combined PDF export is enabled
":" and "/" characters are banned! /:
Traceback (most recent call last):
  File "/usr/bin/mkdocs", line 10, in <module>
    sys.exit(cli())
  File "/usr/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/mkdocs/__main__.py", line 163, in build_command
    ), dirty=not clean)
  File "/usr/lib/python3.6/site-packages/mkdocs/commands/build.py", line 298, in build
    config['plugins'].run_event('post_build', config)
  File "/usr/lib/python3.6/site-packages/mkdocs/plugins.py", line 94, in run_event
    result = method(item, **kwargs)
  File "/usr/lib/python3.6/site-packages/mkdocs_pdf_export_plugin/plugin.py", line 113, in on_post_build
    self.renderer.write_combined_pdf(abs_pdf_path)
  File "/usr/lib/python3.6/site-packages/mkdocs_pdf_export_plugin/renderer.py", line 49, in write_combined_pdf
    self.pages[0].copy(pages).write_pdf(output_path)
AttributeError: 'NoneType' object has no attribute 'copy'
ERROR: 1
```

Full code is here: https://github.com/majkinetor/mm-docs (page demo.md)

**Comment by zhaoterryy** [comment]
Fixed in 0.5.2

**Comment by majkinetor** [comment]
Works now. Thanks
